src/lib/prisma-plugins/n1-detector.ts

```
@@ -0,0 +1,144 @@
+
+    if (!isRelevantOperation || !model) {
+      return next(params);
+    }
+
+    // Consider query arguments for a more precise signature, if needed.
+    // Be cautious with large argument objects in the key.
+    const operationSignature = `${model}.${action}-${JSON.stringify(params.args?.where || {})}`;
+    const now = Date.now();
+    const currentStack = new Error().stack || '';
+
+    let entry = operationCounts.get(operationSignature);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'entry' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)